docs/professions/csv/armorsmithing/recipes.csv

```diff
@@ -0,0 +1,122 @@
+Name,Class,Gender,Level,Materials
+Fire Belt,Peasant,Both,4,5 Linen & 1 Raw Ruby
+Sea Belt,Peasant,Both,4,5 Linen & 1 Raw Sapphire
+Earth Belt,Peasant,Both,4,5 Linen & 1 Raw Beryl
+Wind Belt,Peasant,Both,4,5 Linen & 1 Raw Emerald
+Scout Leather,Rogue,M,8,3 Fine Linen
+Cotte,Rogue,F,8,3 Fine Linen
+Gardcorp,Wizard,M,8,3 Fine Linen
+Magi Skirt,Wizard,F,8,3 Fine Linen
+Cowl,Priest,M,8,3 Fine Linen
+Gorget Gown,Priest,F,8,3 Fine Linen
+Dobok,Monk,M,8,3 Fine Linen
+Earth Bodice,Monk,F,8,3 Fine Linen
+Leather Tunic,Warrior,M,8,3 Fine Linen
+Leather Bliaut,Warrior,F,8,3 Fine Linen
+Leather Gauntlet,Peasant,Both,9,8 Linen
+Leather Beryl Gauntlet,Peasant,Both,11,2 Exquisite Linen & 1 Pristine Beryl
```

Comment:
Instead of using ampersands, can we use commas for consistency with the other recipes? For CSVs, you just need to double quote the string so it doesn't think you mean a new column.